pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -270,8 +270,9 @@ def init_ddp_connection(self, global_rank: Optional[int] = None, world_size: Opt
             torch_distrib.init_process_group(self.torch_distributed_backend, rank=global_rank, world_size=world_size)
 
     def pre_dispatch(self):
-        # move the model to the correct device
-        self.model_to_device()
+        if self.move_to_device_in_prefetch:
+            # move the model to the correct device
+            self.model_to_device()
```

Comment:
would it be easier to override t`pre_disptach` in the FSDP plugin to avoid adding these properties? especially since sync batch norm and configuring ddp are also different